.circleci/config.yml

```diff
@@ -171,6 +171,9 @@ jobs:
     - *restore_js_packages_cache
     - *install_js_packages
     - *save_js_packages_cache
+    - run:
```

Comment:
I was going to suggest putting it in its own job but then I saw how few tests it is! :)